.github/scripts/init-pytest-ui.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "UI pre-run: ensuring pytest accepts charm-file argument"
+
+# Create a temporary conftest.py that adds the --charm-file flag
+cat > tests/conftest.py <<'EOF'
+def pytest_addoption(parser):
+    parser.addoption("--charm-file", action="store", default=None)
+EOF
+
+juju add-model testing || juju switch testing || true
```

Comment:
Why do we need to create the `testing` model? In later steps you apply the ingress-nginx deployment to the `ingress-nginx` namespace.

Reply:
My understanding was that the `charmed-kubernetes/actions-operator` creates a `testing` model automatically on the setup, but they may have removed it or changed it slightly for the `k8s` config? (see [here](https://github.com/charmed-kubernetes/actions-operator/blob/ea90ed489690bf3b1c1fcca6ac5f9edab70aecb0/src/bootstrap/index.ts#L426C1-L434C10)). Can we just confirm that it is not being created anywhere else?